rules/windows/defense_evasion_masquerading_trusted_directory.toml

```diff
@@ -41,13 +42,17 @@ tags = [
     "Data Source: Microsoft Defender for Endpoint",
     "Data Source: Sysmon",
     "Data Source: SentinelOne",
+    "Data Source: Crowdstrike",
 ]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-  process.executable : "C:\\*Program*Files*\\*.exe" and
+  process.executable : (
+    "C:\\*Program*Files*\\*.exe",
```

Comment:
Any reason why there are wildcards?

Reply:
To match directories that masquerade as Program Files, not the original